pkg/k8shandler/kibana/kibana_test.go

```diff
@@ -2,19 +2,18 @@ package kibana
 
 import (
 	"context"
-	"fmt"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	configv1 "github.com/openshift/api/config/v1"
 	consolev1 "github.com/openshift/api/console/v1"
 	routev1 "github.com/openshift/api/route/v1"
-	kibana "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
+	kibanav1 "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
```

Comment:
nit but this would make more sense as `loggingv1` wouldn't it? since we don't have a `kibana` api endpoint...

Reply:
Indeed!